src/psd_tools/api/shape.py

```diff
@@ -164,7 +164,7 @@ class Stroke(object):
 
     def __init__(self, data):
         self._data = data
-        assert self._data.classID == b'strokeStyle'
+        assert self._data.classID == b'strokeStyle' or self._data.classID == b'Strk'
```

Comment:
@Etienne-Gautier Can you do something like the following?

```python
from psd_tools.terminology import Event

if self._data.classID not in (b'strokeStyle', Event.Stroke):
    logger.warning("Unknown class ID found: {}".format(self._data.classID))
```

Reply:
OK not to have a fixture file for now

Reply:
Sure I'll make that change, I've also just managed to get down to a simple version of the file that reproduces the issue (zipped because github doesn't allow PSDs):
[user file cycled.psd.zip](https://github.com/psd-tools/psd-tools/files/10263567/user.file.cycled.psd.zip)